bin/test_e2e

```diff
@@ -2,6 +2,20 @@
 
 source bin/localnet/functions.sh
 
+if [ -n "$TRAVIS_BUILD_NUMBER" ]
```

Comment:
Use something from the list of default travis variables https://docs.travis-ci.com/user/environment-variables/#default-environment-variables